openshift_scalability/config/nodeVertical.yaml

```diff
@@ -4,7 +4,7 @@ projects:
     ifexists: delete
     tuning: default
     pods:
-      - total: 1000
+      - total: 500
```

Comment:
also here -- why is this being done?  the title of the PR is confusing

Reply:
We are returning the default number of pods from 1000 back to 500 so we can run the Node Vertical tests from CI at 250 pods per node and do some NV comparisons of 3.9 vs previous releases.   Was trying to use the same NV config file for CI.  If we want to keep that number at 1000, I could create another config file with 500 and keep it in svt/openshift_scalability/ci/contents/ dir.

Reply:
@jeremyeder @mffiedler PTAL